static/js/lib/ckeditor/plugins/ResourceLinkMarkdownSyntax.ts

```diff
@@ -27,6 +25,17 @@ import {
  * The ResourceEmbed plugin itself is provided via our fork of CKEditor's
  * 'link' plugin.
  */
+
+export const RESOURCE_LINK_SHORTCODE_REGEX = /{{< resource_link (\S+) "(.*?)" >}}/g
+/**
+ * (\S+) to match and capture the UUID
+ * "(.*?)" to match and capture the label text
+ *
+ * Limitations:
+ *   - gets fooled by label texts that include literal `" >}}` values. For
+ *     example, < resource_link uuid123 "silly " >}} link" >}}.
+ */
```

Comment:
I think you might have accidentally moved the link and it's jsdoc comment between the `ResourceLinkMarkdownSyntax` class declaration and it's jsdoc comment here

Reply:
Ahhh silly me. I've moved the regex mostly back up (but put it below the imports)

Reply:
one more thing, and sorry I should have been more clear! but it would be good to put the block comment that you wrote for it above the variable declaration too, like this

```ts
/**
 * Here I can describe
 * What something does or doesn't
 * do through a haiku
 **/
const REGEX = /etc/
```

then various code tools will associate that comment with the `REGEX` variable as a documentation comment - for instance, in my editor if I ask about the type of a variable it will pop up a little window with type information and also print the associated comment, like this:

![Screen Shot 2022-01-24 at 4 39 34 PM](https://user-images.githubusercontent.com/6207644/150869363-7890d087-2886-4acf-98d2-b951790330b1.png)

Reply:
Neat—for some reason I never realized JSDoc comments worked for constants like they do for functions and classes... but why wouldn't they, I suppose.

Reply:
yeah, I don't know if they'll work like that for all documentation tools, but they seem to be well supported w/ typescript ecosystem tools at any rate! super handy to have them pop up when something is confusing me later (usually something I wrote originally haha)